english/wake-up/vigilante/vigilante.md

```diff
@@ -0,0 +1,19 @@
+You slowly open your eyes and found yourself in a dimly lit room, lying on an uncomfortable bed.
+
+You found your right hand is cuffed to the side of the bed.
+
+You also found yourself feeling quite dizzy, you captor must have drugged you after he knocked you out.
+
+Looking for clues, you were investigating a warehouse.
+
+That huge guy came out of no where and start attcking you.
+
+Being the more skillful fighter, you hit him twice and then kicked him in the face.
+
+But that seems to have little effect on him, he counter with a back-hand swing, and hit you on the side of your head.
+
+The next thing you remember is the strange dream of marshmallow and deserted island.
+
+[Lie down](wait/wait.md) again and wait for the effect of the drug to fade...
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.